docs/products/postgresql/howto/use-pg-audit-logging.md

```diff
@@ -0,0 +1,438 @@
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<Tabs>
+<TabItem value="1" label="Aiven Console" groupId="group1" default>
+### Enable in Aiven Console
+
+:::important
+In [Aiven Console](https://console.aiven.io/), you can enable audit logging at the service
+level only. To enable it on a database or for a user, you need to use
+[psql](https://www.postgresql.org/docs/current/app-psql.html).
+:::
+
+1. Log in to [Aiven Console](https://console.aiven.io/), and navigate to your organization
+   \> project > Aiven for PostgreSQL service.
+1. On the **Overview** page of your service, select **Service settings** from the sidebar.
+1. On the **Service settings** page, navigate to the **Advanced configuration** section
+   and select **Configure**.
+1. In the **Advanced configuration** window, select **Add configuration options**, add
+   the `pgaudit.featureEnabled` parameter, set it to `true`, and select
+   **Save configuration**.
+</TabItem>
+<TabItem value="2" label="Aiven API">
+### Enable with Aiven API
+
+You can use the `curl` command line tool to interact with the
+[Aiven API](/docs/tools/api). Call the
+[ServiceUpdate](https://api.aiven.io/doc/#tag/Service/operation/ServiceUpdate) endpoint to
+update your service's configuration by passing `{"pgaudit.featureEnabled": "true"}` in
+the `user_config` object.
+
+```bash
+curl --request PUT                                                                      \
+   --url https://api.aiven.io/v1/project/YOUR_PROJECT_NAME/service/YOUR_SERVICE_NAME    \
+   --header 'Authorization: Bearer YOUR_BEARER_TOKEN'                                   \
+   --header 'content-type: application/json'                                            \
+   --data
+      '{
+         "user_config": {
+            "pgaudit.featureEnabled": "true"
+         }
+      }'
+```
+</TabItem>
+<TabItem value="3" label="Aiven CLI">
+### Enable with Aiven CLI
+
+Use the [Aiven CLI client](/docs/tools/cli) to run the
+[avn service update](/docs/tools/cli/service-cli) command. Update your service by setting
+the `pgaudit.featureEnabled` parameter's value to `true`.
+
+```bash
+avn service update -c pgaudit.featureEnabled=true SERVICE_NAME
+```
+
+:::important
+By default, audit logging does not emit any audit records. To trigger a logging operation
+and start receiving audit records, configure audit logging parameters as detailed in
+[Configure audit logging](#configure-audit-logging).
+:::
+</TabItem>
+<TabItem value="4" label="psql">
+### Enable with psql
+
+:::note
+psql allows for fine-grained enablement of audit logging: on a database, for a user (role),
+or for a database-role combination.
+:::
+
+#### Enable on a database
+
+1. [Connect to your Aiven for PostgreSQL service](/docs/products/postgresql/howto/list-code-samples).
+1. Run the following query:
+
+   ```sql
+   ALTER DATABASE DATABASE_NAME set pgaudit.featureEnabled = 'on'
```

Comment:
`featureEnabled` is not an actual pgaudit setting. We use it internally for knowing if we should install pgaudit or not
```suggestion
   ALTER DATABASE DATABASE_NAME set pgaudit.log = 'ddl'
```